.vscode/settings.json

```diff
@@ -14,7 +14,9 @@
       }
     ]
   },
-  "editor.codeActionsOnSave": { "source.organizeImports": true },
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": "explicit"
+  },
```

Comment:
@Ameer-Alaswad Can we remove this from this PR? It's also happening to me all the time so I will definitely take a look into it, but let's not do something completely unrelated and risk breaking anything for other developers.

You can simply replace "explicit" with "true" or **checkout the file from remote** (keywords for ChatGPT 😂) if you want to do some Git magic